build-system/test-configs/conformance-config.textproto

```diff
@@ -398,3 +398,20 @@ requirement: {
   value: 'DocumentType.prototype.after'
   value: 'CharacterData.prototype.after'
 }
+
+# Document: most of properties should be access via ampdoc for proper scoping.
```

Comment:
@dvoytenko i temporarily added this to trigger the warnings, i'll remove this before merging